tests/functional/test_reinstall_clean.py

```diff
@@ -125,7 +126,7 @@ def test_cylc_install_files(fixture_install_flow, file_, expect):
     [
         (
             'opt/rose-suite-cylc-install.conf', (
-                '# This file records CLI Options.\n\n'
+                '# This file records CLI Options.{version_info}\n'
```

Comment:
This isn't a format string.

The test still passes, so the `version_info` is presumably not require here?